common/postgresql-ng/templates/deployment.yaml

```
@@ -52,6 +52,12 @@ spec:
         env:
         - name: DEBUG
           value: {{ .Values.debug | quote }}
+        - name: PERSISTENCE_ENABLED
+        {{- if .Values.persistence.enabled }}
+          value: true
+        {{- else }}
+          value: false
+        {{- end }}
```

```suggestion
          value: {{ .Values.persistence.enabled }}
```

Sometimes Go templates are less stupid than you think.